.github/workflows/test.yaml

```diff
@@ -0,0 +1,22 @@
+on: [push]
```

Comment:
It turns out `push` only triggers when you *open* a PR (or when you merge), so `pull_request` is more useful here.